pep-0484.txt

```diff
@@ -861,6 +861,73 @@ allow all operations on it, and a value of type ``Any`` can be assigned
 to a variable (or used as a return value) of a more constrained type.
 
 
+Meta-types
+----------
+
+A meta-type can be used to indicate a value that is itself a class
```

Comment:
Following the comment above, I would replace this by "An additional fundamental building block can be used to indicate a value that..." Putting ``Type`` in a row with other building blocks like ``Tuple`` etc.,will  emphasize, that although ``C`` must be a class, ``Type[C]`` is a normal type that can be used elsewhere:
```python
registry = [] # type: List[Tuple[int, Type[FrameworkBase]]]
```